custom_components/alexa_media/alexa_entity.py

```
@@ -68,10 +68,17 @@ def is_known_ha_bridge(appliance: Optional[dict[str, Any]]) -> bool:
         return True
 
-    # If we want to exclude all Matter devices (these can always be added
-    # directly to HA instead of going through AMP), we could test for a
-    # networkInterfaceIdentifier of type "MATTER" or capabilities on the
-    # "Alexa.Matter.NodeOperationalCredentials.FabricManagement" interface.
+    # Identify Matter bridge hubs regardless of manufacturerName
+    if "HUB" in appliance.get("applianceTypes", []):
+        driver_ns = safe_get(appliance, ["driverIdentity", "namespace"], "")
+        driver_id = safe_get(appliance, ["driverIdentity", "identifier"], "")
+        if driver_ns == "AAA" and driver_id == "SonarCloudService":
+            if _has_interface(
```

Pylint (4.0.4)

[error] Undefined variable '_has_interface'

(E0602)